x-pack/plugins/security_solution/public/common/components/filter_group/index.tsx

```diff
@@ -333,10 +324,53 @@ const FilterGroupComponent = (props: PropsWithChildren<FilterGroupProps>) => {
     setShowFiltersChangedBanner(false);
   }, [controlGroup, switchToViewMode, getStoredControlInput, hasPendingChanges]);
 
+  const upsertPersistableControls = useCallback(() => {
+    const persistableControls = initialControls.filter((control) => control.persist === true);
+    if (persistableControls.length > 0) {
+      const currentPanels = controlGroup?.getInput().panels;
+      const orderedPanels = (
+        Object.values(currentPanels ?? []) as Array<ControlPanelState<OptionsListEmbeddableInput>>
```

Comment:
You're running `Object.values` twice on `currentPanels`. Can we pull that out to avariable, or just use `orderedPanels` on line 336?

Reply:
Done. Thanks